lib/data.js

```diff
@@ -0,0 +1,66 @@
+const data = [
+  {
+    id: "1",
+    title: "Buy groceries",
+    priority: "High",
+    dueDate: "2024-09-15",
+  },
+  {
+    id: "2",
+    title: "Finish project report",
+    priority: "Medium",
+    dueDate: "2024-08-26",
+  },
+  {
+    id: "3",
+    title: "Gym workout",
+    priority: "Low",
+    dueDate: "2024-07-12",
+  },
+  {
+    id: "4",
+    title: "Call plumber",
+    priority: "High",
+    dueDate: "2024-10-14",
+  },
+  {
+    id: "5",
+    title: "Schedule dentist appointment",
+    priority: "Medium",
+    dueDate: "2024-12-16",
+  },
+  {
+    id: "6",
+    title: "Prepare presentation",
+    priority: "High",
+    dueDate: "2024-09-17",
+  },
+  {
+    id: "7",
+    title: "Plan weekend trip",
+    priority: "Low",
+    dueDate: "2024-09-18",
+  },
+  {
+    id: "8",
+    title: "Organize closet",
+    priority: "Medium",
+    dueDate: "2024-09-19",
+  },
+  {
+    id: "9",
+    title: "Read a book",
+    priority: "Low",
+    dueDate: "2024-10-20",
+  },
+  {
+    id: "10",
+    title: "Update software",
+    priority: "Medium",
+    dueDate: "2024-12-21",
+  },
+];
+
+export default function getAllTasks() {
+  return data;
+}
```

Comment:
You do not need this function. You can simply export the data variable and import them in your `ToDos.js`.

```suggestion
```

The import would look something like this:
```
import { initialToDos } from "./lib/data.js";